cppcache/src/TcrMessage.cpp

```diff
@@ -1530,7 +1530,7 @@ void TcrMessage::handleByteArrayResponse(
           }
           m_metadata->push_back(bucketServerLocations);
         }
-        LOGFINER("Metadata size is %", m_metadata->size());
+        LOGFINER("Metadata size is %d", m_metadata->size());
```

Comment:
I believe the `%zu` is what you want for `size_t` values.

Reply:
Good catch.